lib/dnsimple/error.rb

```diff
@@ -9,8 +9,19 @@ class RequestError < Error
 
     def initialize(http_response)
       @http_response = http_response
-      super(http_response.code.to_s)
+      super(message_from(http_response))
     end
+
+    private
+
+    def message_from(http_response)
+      if http_response.headers["Content-Type"] == "application/json"
+        http_response.parsed_response["message"]
+      else
+        http_response.code
```

Comment:
I'm fine with that.

FYI I believe Net:HTTP has a hash of codes => messages, we can reuse if we want. We already pull in the deps with HTTParty, and it's stdlib, hence it would be fine.